src/styles/blocks/page.scss

```diff
@@ -0,0 +1,18 @@
+body {
+  font-family: "Roboto", "Arial", sans-serif;
+
+  --main-accent: #060b35;
```

Comment:
SASS uses a separate file for variables. Learn more [here](https://mate.academy/learn/css-advanced/sass?section=video&videoId=4949).